phi/tools/video.py

```diff
@@ -0,0 +1,132 @@
+from phi.tools import Toolkit
+from moviepy import VideoFileClip, TextClip, CompositeVideoClip  # type: ignore
```

Comment:
Just add a check and recommendation to install moviepy if it can't be imported. Like we have with other tools

Reply:
done